collect_app/src/main/java/org/odk/collect/android/preferences/GeneralPreferencesFragment.java

```diff
@@ -79,7 +80,7 @@ public boolean onPreferenceClick(Preference preference) {
                     break;
             }
             if (basePreferenceFragment != null) {
-                getActivity().getFragmentManager()
+                getActivity().getSupportFragmentManager()
```

Comment:
Now we're doing this I think we should be able to get rid of the `AndroidXPreferencesActivity` and just start those fragments like everything else. Could you make that happen please?